tests/cnf/ran/diskencryption/tests/tpm2.go

```diff
@@ -0,0 +1,358 @@
+package tests
+
+import (
+	"context"
+	"regexp"
+	"time"
+
+	"github.com/golang/glog"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/internal/file"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/internal/helper"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/internal/parsehelper"
+	stdinmatcher "github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/internal/stdin-matcher"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/tsparams"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/cluster"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+)
+
+var _ = Describe("TPM2", func() {
+	var (
+		nodeList []*nodes.Builder
```

Comment:
I missed this, but if nodeList is only used in BeforeEach it can just be declared closer to where it is used

Reply:
fixed